templates/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+require "simplecov"
+SimpleCov.start "rails"
```

Comment:
should this be in `rails_helper`  that's where I've seen it in a recent project that used SimpleCov. (although it looks like we were almost always requiring `rails_helper` rather than `spec_helper`, which is not ideal)